### PR TITLE
[CI] Pipeline migration master

### PR DESCRIPTION
**Pull request by liushilongbuaa**
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

**Comment by liushilongbuaa** [comment]
The former PR is invalid, because of re-forking repo of sonic-buildimage.

**Comment by liushilongbuaa** [comment]
Changed from multi-stages pipeline to single stage pipelines.

**Review by lguohan — DISMISSED** [review]

File: .azure-pipelines/official-images/official-build-barefoot.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Starter pipeline
```
Comment (xumia) [inline .azure-pipelines/official-images/official-build-barefoot.yml:1]: You can use the same yaml file for all build definitions, different definition uses different parameters.

Comment (liushilongbuaa) [reply .azure-pipelines/official-images/official-build-barefoot.yml:1]: Maybe I can drop another PR for compare. 
This kind of change will remove template parameters limit, is hard to set default values, and is like classic Jenkins job. parameters are in Web config, not in code. I prefer not use the same yaml file.

**Pull request closed without merging**